mudlet-mapper.xml

```diff
@@ -4681,7 +4681,9 @@ function createOption(startingValue, onChangeFunc, allowedVarTypes, use)
 		value = startingValue,
 		onChange = onChangeFunc,
 		allowedVarTypes = allowedVarTypes,
-		use = use or &quot;&quot;
+		use = use or &quot;&quot;,
+      checkOption = checkOption or function() return true end
```

Comment:
Indentation is funny here (most places use tabs, not spaces... We should change that at some point)